pkg/services/sqlstore/org_users_test.go

```diff
@@ -0,0 +1,176 @@
+package sqlstore
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/grafana/pkg/models"
+)
+
+type getOrgUsersTestCase struct {
+	desc             string
+	query            *models.GetOrgUsersQuery
+	expectedErr      error
+	expectedNumUsers int
+}
+
+func TestSQLStore_GetOrgUsers(t *testing.T) {
+	tests := []getOrgUsersTestCase{
+		{
+			desc: "should return all users",
+			query: &models.GetOrgUsersQuery{
+				OrgId: 1,
+				User: &models.SignedInUser{
+					OrgId:       1,
+					Permissions: map[int64]map[string][]string{1: {"org.users:read": {"users:*"}}},
+				},
+			},
+			expectedNumUsers: 10,
+		},
+		{
+			desc: "should return no users",
+			query: &models.GetOrgUsersQuery{
+				OrgId: 1,
+				User: &models.SignedInUser{
+					OrgId:       1,
+					Permissions: map[int64]map[string][]string{1: {"org.users:read": {""}}},
+				},
+			},
+			expectedNumUsers: 0,
+		},
+		{
+			desc: "should return some users",
+			query: &models.GetOrgUsersQuery{
+				OrgId: 1,
+				User: &models.SignedInUser{
+					OrgId: 1,
+					Permissions: map[int64]map[string][]string{1: {"org.users:read": {
+						"users:id:1",
+						"users:id:5",
+						"users:id:9",
+					}}},
+				},
+			},
+			expectedNumUsers: 3,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.desc, func(t *testing.T) {
+			store := InitTestDB(t)
```

Comment:
nit: I think you can more InitTestDB outside of the for to improve test performance